validator/client/sync_committee.go

```diff
@@ -89,6 +89,11 @@ func (v *validator) SubmitSyncCommitteeMessage(ctx context.Context, slot types.S
 		"validatorIndex":     msg.ValidatorIndex,
 	}).Info("Submitted new sync message")
 	atomic.AddUint64(&v.syncCommitteeStats.totalMessagesSubmitted, 1)
+
+	fmtKey := fmt.Sprintf("%#x", pubKey[:])
+	if v.emitAccountMetrics {
+		ValidatorSyncCommitteeGaugeVec.WithLabelValues(fmtKey).Set(float64(slot))
+	}
```

Comment:
I think there is a few problems with this approach.

1) This sets the value to a slot. I would assume, based on the description, that the gauge value would be a boolean 0 or 1.
2) This doesn't reset or unset the gauge when the validator stops being part of a current sync committee.
3) This doesn't fill all of your validator keys in the gauge. Same problem as number 2. 

For the third point, that may be desired, but I'm not sure. 

I think that we need to add more information to the function that reports validator performance metrics